docs/features/software-catalog/descriptor-format.md

```diff
@@ -654,7 +654,7 @@ spec:
   steps:
     - id: fetch-base
       name: Fetch Base
-      action: fetch:cookiecutter
+      action: fetch:template
```

Comment:
This PR updates examples of `template.yaml` throughout the docs to refer to `fetch:template` instead of `fetch:cookiecutter`. We can also split this up so that integrators are pointed towards the new action a bit more slowly. WDYT @OrkoHunter @benjdlambert?

Reply:
Docs are fine to update right away I'd say. Probably not example templates though which are ingested via URL by people who have not yet upgraded.

Reply:
Makes sense to me! Our plan with the example templates is to leave the existing ones in place and introduce the new ones that use `fetch:template` in the `software-templates` repository in https://github.com/backstage/backstage/issues/6286.